ansible/roles/test/tasks/bgp_gr_helper/routes_update_expect.txt

```diff
@@ -0,0 +1 @@
+r, ".*Create route.*"
```

Comment:
Can you please describe what does this rule describe?

Reply:
When normal reboot is performed on VM, the peer device (SONiC) is not supposed to preserve the routes. After the VM is up, SONiC should learn BGP routes from VM again. This kind of logs are expected in /var/log/syslog:

Jul 18 07:21:24.264802 mtbc-sonic-01-2410 INFO swss#orchagent: :- addRoute: Create route 192.168.14.0/32 with next hop(s) 10.0.0.1,10.0.0.5,10.0.0.9,10.0.0.13,10.0.0.21,10.0.0.25,10.0.0.29
Jul 18 07:21:24.265080 mtbc-sonic-01-2410 INFO swss#orchagent: :- addRoute: Create route 192.168.14.1/32 with next hop(s) 10.0.0.1,10.0.0.5,10.0.0.9,10.0.0.13,10.0.0.21,10.0.0.25,10.0.0.29
Jul 18 07:21:24.265364 mtbc-sonic-01-2410 INFO swss#orchagent: :- addRoute: Create route 192.168.14.112/32 with next hop(s) 10.0.0.1,10.0.0.5,10.0.0.9,10.0.0.13,10.0.0.21,10.0.0.25,10.0.0.29

Reply:
So here you expect to see only "Create route" messages.

Reply:
Yes, the exact logic here is that: if pattern ".*Create route.*" is not observed in syslog, fail the case.